src/standard-live-connect.js

```diff
@@ -41,18 +41,19 @@ import { resolve as decisionsResolve } from './manager/decisions'
 import { enrich as pageEnrich } from './enrichers/page'
 import { enrich as identifiersEnrich } from './enrichers/identifiers'
 import { enrich as privacyConfig } from './enrichers/privacy-config'
-import { isArray, isObject, merge } from './utils/types'
+import { isArray, isFunction, isObject, merge } from './utils/types'
 import { IdentityResolver } from './idex/identity-resolver'
 import { StorageHandler } from './handlers/storage-handler'
 import { CallHandler } from './handlers/call-handler'
 import { StorageStrategy } from './model/storage-strategy'
+import ConfigManager, { qualifiedConfig } from './config/config-manager'
 
 const hemStore = {}
-function _pushSingleEvent (event, pixelClient, enrichedState) {
+function _pushSingleEvent (event, pixelClient, enrichedState, configManager) {
   if (!event || !isObject(event)) {
     emitter.error('EventNotAnObject', 'Received event was not an object', new Error(event))
   } else if (event.config) {
-    emitter.error('StrayConfig', 'Received a config after LC has already been initialised', new Error(event))
+    configManager.push(event.config)
```

Comment:
for those who use npm module, in order to allow multiple configuration, here in this part should we pick the best configuration and send the event to pixelClient?

Reply:
The NPM module is not meant to be configured via window.liQ.push. Instead, the config must be passed explicitly when calling the create function. The code here collects the configs that arrive via window.liQ.push because it may be required by the LC initialized in the next step. Specifically, it covers this case:

- init NPM module (the config is provided explicitly)
- window.liQ.push({config: {...}})
- load builder script